sql/src/main/java/io/crate/analyze/CopyAnalyzer.java

```diff
@@ -240,16 +240,23 @@ CopyToAnalyzedStatement convertCopyTo(CopyTo node, Analysis analysis) {
             throw new UnsupportedFeatureException("Output format not supported without specifying columns.");
         }
 
-        QuerySpec querySpec = new QuerySpec()
-            .outputs(outputs)
-            .where(createWhereClause(
+        QuerySpec querySpec = new QuerySpec(
+            outputs,
+            createWhereClause(
                 node.whereClause(),
                 partitions,
                 normalizer,
                 expressionAnalyzer,
                 expressionAnalysisContext,
-                analysis.transactionContext())
-            );
+                analysis.transactionContext()
+            ),
+            List.of(),
+            null,
+            null,
+            null,
+            null,
+            false
```

Comment:
this is quite hard to read/understand (esp in non ide envs) - would it be an option to use a builder class for the QuerySpec?

Reply:
Given that my plan was to move away from the QuerySpec I'd rather avoid adding a builder which promotes using it.

I've already some ideas to get rid of this here again by avoiding using `QueriedTable` - but I'm still exploring a few different options.